prow/plugins/lgtm/lgtm.go

```diff
@@ -56,11 +59,11 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 		Description: "The lgtm plugin manages the application and removal of the 'lgtm' (Looks Good To Me) label which is typically used to gate merging.",
 	}
 	pluginHelp.AddCommand(pluginhelp.Command{
-		Usage:       "/lgtm [cancel] or Github Review action",
-		Description: "Adds or removes the 'lgtm' label which is typically used to gate merging.",
+		Usage:       "/lgtm [cancel] or /tbr or Github Review action",
+		Description: "Adds or removes the 'lgtm' label which is typically used to gate merging. If enabled, /tbr also adds 'to-be-reviewed' label",
```

Comment:
Can't we determine if it is or is not enabled here, and serve appropriate help?

Reply:
do we have an example of this? SGTM

Reply:
@cjwagner ?